src/main/java/Duke.java

```diff
@@ -1,10 +1,331 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Arrays;
+import java.io.File;
+import java.io.FileWriter;
+
 public class Duke {
+
+    /**
+     * Array of tasks
+     */
```

Comment:
This comment seems a bit redundant because we can easily tell that `tasks` is an arraylist of tasks since you have named it appropriately and from its data type.